File: simulation/src/sim/MovementVars.java
package sim;

public class MovementVars {
    public static double movement_x = 0;
    public static double movement_y = 0;
    public static double movement_turn = 0;

    public static void stopMovement() {
        movement_x = 0;
        movement_y  = 0;
        movement_turn = 0;
    }
}

File: simulation/src/sim/MovementVars.kt
package sim

import sim.MovementVars

object MovementVars {
    var movement_x = 0.0
    var movement_y = 0.0
    var movement_turn = 0.0
    fun stopMovement() {
        movement_x = 0.0
        movement_y = 0.0
        movement_turn = 0.0
    }
}